components/Modal/CreateImageUpload.js

```diff
@@ -0,0 +1,751 @@
+import React from "react";
+import {
+  Button,
+  Form,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+  Checkbox,
+  TextContent,
+  Text,
+  TextInput,
+  TextList,
+  TextListVariants,
+  TextListItem,
+  TextListItemVariants,
+  Title,
+  Wizard,
+  WizardContextConsumer,
+  WizardFooter
+} from "@patternfly/react-core";
+import { defineMessages, FormattedMessage, injectIntl, intlShape } from "react-intl";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import NotificationsApi from "../../data/NotificationsApi";
+import BlueprintApi from "../../data/BlueprintApi";
+import { setBlueprint } from "../../core/actions/blueprints";
+import { fetchingQueue, clearQueue, startCompose, fetchingComposeTypes } from "../../core/actions/composes";
+import { fetchingUploadProviders } from "../../core/actions/uploads";
+
+const messages = defineMessages({
+  infotip: {
+    defaultMessage: "This process can take a while. " + "Images are built in the order they are started."
+  },
+  warningUnsaved: {
+    defaultMessage:
+      "This blueprint has changes that are not committed. " +
+      "These changes will be committed before the image is created."
+  },
+  selectOne: {
+    defaultMessage: "Select one"
+  },
+  title: {
+    defaultMessage: "Create and Upload Image"
```

Comment:
The modal title is 'Create and Upload Image', however for many image types upload operation is not supported.

Reply:
Do you think we should start the title as Create Image and then if the user selects an upload provider change it to Create and Upload Image?